codalab/lib/bundle_cli.py

```diff
@@ -1034,7 +1034,10 @@ def auto_convert_type(value):
 
     @Commands.command(
         'workers',
-        help=['Display worker information of this CodaLab instance. Root user only.'],
+        help=[
+            'Display worker information of this CodaLab instance. '
+            'If not root user, only display information about your connected workers.'
```

Comment:
Change: "Display information about workers that you have connected to the CodaLab instance." (most people won't need to worry about the root user).